lib/Doctrine/DBAL/Types/Exception/ValueNotConvertible.php

```diff
@@ -20,7 +20,7 @@ public static function new($value, $toType, ?string $message = null) : self
         if ($message !== null) {
             return new self(
                 sprintf(
-                    "Could not convert database value to '%s' as an error was triggered by the unserialization: '%s'",
+                    'Could not convert database value to "%s" as an error was triggered by the unserialization: "%s".',
```

Comment:
Quotes are not needed for the last parameter (chained error message). The period is probably okay since the error message will come from PHP, not from us.